### PR TITLE
Fix a few bugs for nested collections

### PR DESCRIPTION
**Pull request by wachterjohannes**
@danrot this PR is related to the list of bugs which i send to you

### CODE REVIEW

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/root/main.js [hunk 1]
```diff
@@ -89,13 +89,13 @@ define(function() {
...
             this.sandbox.on('sulu.list-toolbar.change.thumbnail-small', function() {
-                this.sandbox.emit('husky.datagrid.view.change', 'thumbnail', listViews['thumbnailSmall']);
+                this.sandbox.emit('husky.datagrid.view.change', 'thumbnail', listViews['thumbnailSmall']['thViewOptions']);
                 this.sandbox.sulu.saveUserSetting(constants.listViewStorageKey, 'thumbnailSmall');
             }.bind(this));
 
             // change datagrid to thumbnail large
             this.sandbox.on('sulu.list-toolbar.change.thumbnail-large', function() {
-                this.sandbox.emit('husky.datagrid.view.change', 'thumbnail', listViews['thumbnailLarge']);
+                this.sandbox.emit('husky.datagrid.view.change', 'thumbnail', listViews['thumbnailLarge']['thViewOptions']);
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/root/main.js]: Where is this value set? And we don't use abbreviations...

Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/root/main.js]: the datagrid needs this name so when we dont want to change it there it has to be this name (=

Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/root/main.js]: this value is set on a variable in this file ...

File: src/Sulu/Bundle/MediaBundle/Api/Collection.php [hunk 2]
```diff
@@ -31,14 +31,14 @@
...
- *          parameters = { "collection" = "expr(object.getId())", "limit" = 9999  }
+ *          parameters = { "collection" = "expr(object.getId())", "limit" = 9999, "orderBy" = "media.changed", "orderSort" = "DESC" }
  *      )
  * )
  * @Relation(
  *      "filterByTypes",
  *      href = @Route(
  *          "cget_media",
- *          parameters = { "collection" = "expr(object.getId())", "limit" = 9999, "types" = "{types}" }
+ *          parameters = { "collection" = "expr(object.getId())", "limit" = 9999, "types" = "{types}", "orderBy" = "media.changed", "orderSort" = "DESC" }
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Api/Collection.php]: Shouldn't these parameters be removed as well? (same changes above)

Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Api/Collection.php]: fixed

**Pull request merged**